tests/installation/ipxe_install.pm

```diff
@@ -203,6 +273,11 @@ sub run {
 
     select_console 'sol', await_console => 0;
 
+    if (is_disk_image) {
+        check_screen([qw(load-linux-kernel load-initrd)], 120 / get_var('TIMEOUT_SCALE', 1));
```

Comment:
What is the point of check- screen if we are not validating screen it at all?

Reply:
linx/initrd loading always being checked in this module for other installations, so I just keep it. I think it worth checking because linux/initrd loading is the first step or beginning of ipxe installation.